package.json

```diff
@@ -101,6 +112,7 @@
   "dependencies": {
     "execa": "0.5.0",
     "lru-cache": "4.0.2",
+    "mz": "2.6.0",
```

Comment:
What do you need mz for? vscode is running with node6 which supports most es2015/es2016 features and asyc functions are enabled through typescript.